smqtk_iqr/utils/mongo_sessions.py

```diff
@@ -10,35 +10,41 @@
 from datetime import datetime, timedelta
 import logging
 from uuid import uuid4
+from typing import Callable, Union, Iterable, Tuple, Mapping, TypeVar, Optional, Any
 
+import flask
 from flask.sessions import SessionInterface, SessionMixin
+from flask.wrappers import Request, Response
 from werkzeug.datastructures import CallbackDict
 from pymongo import MongoClient  # type: ignore
 
 
 LOG = logging.getLogger(__name__)
 
+T = TypeVar("T", bound="MongoSession")
 
 class MongoSession(CallbackDict, SessionMixin):
 
-    def __init__(self, initial=None, sid=None):
+    def __init__(
+        self, initial: Union[Mapping, Iterable[Tuple[Any, Any]], None] = None,
+        sid: Optional[str] = None):
         super(MongoSession, self).__init__(initial, MongoSession.on_update)
```

Comment:
`error: Argument 2 to "__init__" of "CallbackDict" has incompatible type "Callable[[MongoSession], None]"; expected "Optional[Callable[[CallbackDict[Any, Any]], None]]`
Unsure of what exactly the issue is here